kubernetes/manifests/pod.yml.j2

```diff
@@ -0,0 +1,84 @@
+{% set app_name = "screenshot" -%}
+{% set service_port = "3002" %}
+{% set resources = {"requests": {"cpu": "1", "mem": "2G"}, "limits": {"cpu": "1", "mem": "2G"} } %}
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ app_name }}
+  namespace: {{ K8S_ENV }}
+  annotations:
+    git_commit: {{ BUILD_GIT_COMMIT }}
+    git_branch: {{ BUILD_GIT_BRANCH }}
+spec:
+  revisionHistoryLimit: 3
+  minReadySeconds: 5
+  strategy:
+    rollingUpdate:
+      maxSurge: 15%
+      maxUnavailable: 0
+    type: RollingUpdate
+  template:
+    metadata:
+      annotations:
+        git_commit: {{ BUILD_GIT_COMMIT }}
+        git_branch: {{ BUILD_GIT_BRANCH }}
+      labels:
+        app: {{ app_name }}
+        environment: {{ K8S_ENV }}
+    spec:
+      nodeSelector:
+        cloud.google.com/gke-nodepool: {{ K8S_NODEPOOL }}
+      containers:
+        - name: {{ app_name }}
+          command: ["/usr/bin/service-entrypoint.sh"]
+          image: gcr.io/netlify-services/{{ app_name }}
+          imagePullPolicy: Always
+          env:
+            - name: K8S_PROVIDER
+              value: {{ K8S_PROVIDER }}
+            - name: K8S_CLUSTER
+              value: {{ K8S_CLUSTER }}
+            - name: K8S_ENV
+              value: {{ K8S_ENV }}
+            - name: SERVICE_PORT
+              value: "{{ service_port }}"
```

Comment:
this is not used inside the container right now

Reply:
Changed to PORT, which is the env variable used by node code